src/System.Private.Uri/tests/System.Private.Uri.Tests.builds

```diff
@@ -16,6 +16,16 @@
       <TestTFMs>netcore50;netcoreapp1.0;net46</TestTFMs>
       <OSGroup>Windows_NT</OSGroup>
     </Project>
+    <Project Include="ExtendedFunctionalTests\System.Private.Uri.ExtendedFunctional.Tests.csproj">
+      <TargetGroup>netstandard1.7</TargetGroup>
```

Comment:
Discussed offline. We agreed that netstandard1.7 is the right TargetGroup

---
In reply to: [72678399](https://github.com/dotnet/corefx/pull/10297#discussion_r72678399) [](ancestors = 72678399)